File: day3/armstrong.cpp
#include <iostream>
using namespace std;

int main()
{
    int num;
    cout << "Enter your number" << endl;
    cin >> num;
    int initial = num;
    int final = 0;
    while (num > 0)
    {
        int lastdigit = num % 10;
        final = final + (lastdigit * lastdigit * lastdigit);
        num = num / 10;
    }
    if (final == initial)
    {
        cout << "This number is Armstrong number";
    }
    else
    {
        cout << "This number is not Armstrong number";
    }
    return 0;
}

File: day11/armstrong.java
package day11;

import java.util.Scanner;

public class armstrong {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter your number");
        int num = sc.nextInt();
        int initial = num;
        int arm = 0;
        while(num != 0){
            int divide = num%10;
            arm += (divide*divide*divide);
            num = num/10;
        }
if(initial == arm){
System.out.println("This is armstrong number");
}else{
    System.out.println("This is not armstrong number");
}
    }
}
